### PR TITLE
Improve help and doc #22

### PR DESCRIPTION
**Pull request by pombredanne**
This PR improves the intro documentation in the readme and the generated website help page.

Reference: https://github.com/nexB/scancode-licensedb/issues/22

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,126 @@
+ScanCode LicenseDB
```
Comment (pombredanne) [inline README.rst:1]: @mjherzog @DennisClark FYI... this parts would need your review and I could then propagate the edits to the help page too

**Review by AyanSinhaMahapatra — APPROVED** [review]
I've pushed a commit with some minor changes:

* update reference to .LICENSE files correctly
* update README with the same and misc
* add new CLA license category
* update license category docs

**Comment by pombredanne** [comment]
@AyanSinhaMahapatra ^ not sure how this meshes with your work in progress in ScanCode toolkit

**Comment by AyanSinhaMahapatra** [comment]
@pombredanne I've had pushed a commit here and also updated this in #26 so it is all updated AFAIK

**Comment by AyanSinhaMahapatra** [comment]
No we missed the help.html which got replaced as we now build from scancode-toolkit. Opened a PR to fix this.

**Pull request merged**